src/App/PackageService.php

```diff
@@ -104,16 +104,20 @@ public function gitSetUpstream(Package $package, OutputManager $io): void
         }
     }
 
-    public function createSymbolicLinks(PackageList $packageList, OutputManager $io): void
+    /**
+     * @param Package[] $targetPackages
+     * @param OutputManager $io
+     * @return void
+     */
+    public function createSymbolicLinks(array $targetPackages, OutputManager $io): void
     {
         $io
             ->important()
             ->info('Re-linking vendor directories...');
 
-        $installedPackages = $packageList->getInstalledAndEnabledPackages();
-        foreach ($installedPackages as $package) {
+        foreach ($targetPackages as $package) {
             $io->info("Package <package>{$package->getId()}</package> linking...");
-            $this->linkPackages($package, $installedPackages);
+            $this->linkPackages($package, $targetPackages);
```

Comment:
This code linkage target packages itself only. Need linkage target packages in all dependent packages.

For example, taget package `yiisoft/strings` only. Need to update links in all packages that depended from `yiisoft/strings` (`yiisoft/files`, `yiisoft/http`, ...).